src/Responses/GetMeetingsResponse.php

```diff
@@ -27,23 +27,19 @@
  */
 class GetMeetingsResponse extends BaseResponse
 {
-    /**
-     * @var Meeting[]
-     */
-    private $meetings;
-
     /**
      * @return Meeting[]
      */
-    public function getMeetings()
+    public function getMeetings(): array
     {
-        if (null === $this->meetings) {
-            $this->meetings = [];
-            foreach ($this->rawXml->meetings->children() as $meetingXml) {
-                $this->meetings[] = new Meeting($meetingXml);
+        $meetings = [];
```

Comment:
Trade-off between more memory (old version) and more cpu (new version). Need a good argument to decided which solution to choose for the community. We assume the XML content is static and never changes in a single API call.

Reply:
Technically you are right (memory vs. cpu), but I guess those API-small data do no harm to any of them in these days. This proposal is just a cleaner and leaner way ... and it is more robust against programming errors (less code, less variables, low risk of inconsistent data).